### PR TITLE
New ArcGIS notebook to validate item metadata

### PR DESCRIPTION
**Pull request by ManushiM**
This is an ArcGIS Notebook available under Esri sample Notebooks in AGOL. Refer to this epic in [ArcGIS:geosaurus#9824](https://github.com/ArcGIS/geosaurus/issues/9824) to learn more.

For this notebook:
1. I tested to see if it works.
2. I reworded the markdown cells a little and replaced the url for `Enterprise Metadata documentation`. 

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? 
    - [ ] First block of imports are standard libraries
    - [ ] Second block are 3rd party libraries
    - [ ] Third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('home')` or `gis = GIS('pro')`
    - `gis = GIS(profile="your_online_portal")`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the `api_data_owner` user?
- [ ] If the notebook requires working with local data (such as CSV, FGDB, SHP, Raster files), upload the files as items to the [Geosaurus Online Org](geosaurus.maps.arcgis.com/) using `api_data_owner` account and change the notebook to first download and unpack the files.
- [ ] Code simplified & split out across multiple cells, useful comments?
- [ ] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [ ] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] Is your code formatted using [Jupyter Black](https://www.freecodecamp.org/news/auto-format-your-python-code-with-black/)? You can use Jupyter Black to format your code in the  notebook.
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ mohi9282 so he can add it to the list for the next deploy

### CODE REVIEW

File: samples/03_org_administrators/validate_item_metadata.ipynb [hunk 1]
```diff
@@ -0,0 +1,385 @@
+{
```
Comment (CMPeng) [inline samples/03_org_administrators/validate_item_metadata.ipynb:1]: <p>Thanks Manushi for adding this notebook!</p><p>Looking good to me except that I cannot view the output display of DataFrame or a list of sample <code>metadata</code> output.</p><p>Also, I believe the term <code>metadata</code> refers to this metadata property of the item dict. Can we either make some clarification that the metadata we are discussing here is not the same concept, or can we add one more column to the output data frame that also describes the <code>metadata</code> of a web item?    </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1566/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (cariashuang0417) [inline samples/03_org_administrators/validate_item_metadata.ipynb:1]: <p>We can also add a script to delete the item at the end of this notebook (though it doesn't create any service)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1566/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (ManushiM) [reply samples/03_org_administrators/validate_item_metadata.ipynb:1]: Thanks @CMPeng ! I updated the notebook to display first 5 rows of that dataframe and first 5 items of the `item_profile_status` dictionary. That way the user won't see everything happening in the geosaurus org. ;)

I also added a clarification for `metadata` in the first markdown cell, after the link to Enterprise Metadata documentation. Let me know if it needs any changes.

Comment (ManushiM) [reply samples/03_org_administrators/validate_item_metadata.ipynb:1]: Thanks @cariashuang0417 . I added the last `item.delete()` cell with a markdown before it to explain it.

**Review by CMPeng — APPROVED** [review]
The updates look good to me! Thanks Manushi~

**Pull request merged**